jobdsl/organisations-beta.groovy

```diff
@@ -10,6 +10,7 @@ private boolean isSandbox() {
 List<Map> orgs = [
         [name: 'CNP'],
         [name: 'PCQ', regex: 'pcq-.*'],
+        [name: 'ADOPTION', regex: 'adoption-.*'],
```

Comment:
You probably don't want capitals for a word, and the regex will use the name by default, (yes pcq should be changed to, missed during review)
```suggestion
        [name: 'Adoption'],
```

Reply:
thanks @timja. Now Updated